web_widget_image_download/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+===========================
+Web Widget - Image Download
+===========================
+
+This module was written to extend the functionality of the image widget and allows to download it.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to the section `Contacts`.
+#. Click on a contact.
+#. Edit the contact.
+#. Click the `Download` button (between `Edit` and `Clear`).
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/162/8.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/web/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Images
```

Comment:
Yeah, but you can make another PR to amend it. There are 2 more things that I have thought about this weekend:

* If there's no image, is there the button to download?
* If the image is not png, the extension shouldn't be png (for example, jpg or gif). We should detect the type. Is there an easy way for that?